main.go

```diff
@@ -6,16 +6,22 @@ package main
 
 import (
 	"context"
-
-	"github.com/owner/name/plugin"
-
+	"github.com/drone/drone-nuget/plugin"
+	"github.com/joho/godotenv"
```

Comment:
imports are generally organized in groups, see https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Cheers, I've now fixed this.